src/Repositories/Body/ArrayBodyRepository.php

```diff
@@ -57,39 +63,40 @@ public function merge(array ...$arrays): static
     /**
      * Add an element to the repository.
      *
-     * @param string $key
-     * @param mixed $value
+     * @param string|int|null $key
+     * @param mixed|null $value
      * @return $this
      */
-    public function add(string $key, mixed $value): static
+    public function add(string|int|null $key = null, mixed $value = null): static
     {
-        $this->data[$key] = $value;
+        isset($key)
+            ? $this->data[$key] = $value
+            : $this->data[] = $value;
 
         return $this;
     }
 
     /**
-     * Remove an item from the repository.
+     * Get a specific key of the array
      *
-     * @param string $key
-     * @return self
+     * @param string|int $key
+     * @param mixed|null $default
+     * @return mixed
      */
-    public function remove(string $key): static
+    public function get(string|int $key, mixed $default = null): mixed
     {
```

Comment:
Since `ArrayBodyRepository` supports adding elements without a key/index, basically acting as a list, would it make sense to also use a nullable key and return the whole body if none was provided?

```suggestion
    public function get(string|int|null $key = null, mixed $default = null): mixed
    {
        if (null === $key) {
            return $this->all();
        }
```